src/consts.rs

```diff
@@ -0,0 +1,4 @@
+//! Constants used in the builder.
```

Comment:
mega nit: let's just call this file `constants.rs` since that's the usual name elsewhere